README.md

```diff
@@ -1,6 +1,5 @@
 
-
-# Project Name
+# Hello Microverse
```

Comment:
- [Optional] Please check your LinkedIn social link, it links to someone else page on LinkedIn, please fix it